modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/EmailNotificationTypeTest.java

```diff
@@ -1405,6 +1394,60 @@ private ObjectAction _addObjectAction(
 			false);
 	}
 
+	private ObjectDefinition _addObjectDefinitionWithObjectAction(Role role)
```

Comment:
Could you rename to `_addObjectDefinitionWithNotificationTemplateObjectAction` ?